Source/DataSources/GpxDataSource.js

```diff
@@ -0,0 +1,1223 @@
+/*global define*/
+define([
+        '../Core/BoundingRectangle',
+        '../Core/Cartesian2',
+        '../Core/Cartesian3',
+        '../Core/ClockRange',
+        '../Core/ClockStep',
+        '../Core/Color',
+        '../Core/createGuid',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/DeveloperError',
+        '../Core/Ellipsoid',
+        '../Core/Event',
+        '../Core/getFilenameFromUri',
+        '../Core/Iso8601',
+        '../Core/JulianDate',
+        '../Core/loadBlob',
+        '../Core/loadXML',
+        '../Core/Math',
+        '../Core/NearFarScalar',
+        '../Core/PinBuilder',
+        '../Core/PolygonHierarchy',
+        '../Core/Rectangle',
+        '../Core/RuntimeError',
+        '../Core/TimeInterval',
+        '../Core/TimeIntervalCollection',
+        '../Scene/HorizontalOrigin',
+        '../Scene/LabelStyle',
+        '../ThirdParty/Autolinker',
+        '../ThirdParty/Uri',
+        '../ThirdParty/when',
+        '../ThirdParty/zip',
+        './BillboardGraphics',
+        './CompositePositionProperty',
+        './ConstantPositionProperty',
+        './DataSource',
+        './DataSourceClock',
+        './Entity',
+        './EntityCollection',
+        './LabelGraphics',
+        './PathGraphics',
+        './PointGraphics',
+        './PolygonGraphics',
+        './PolylineGraphics',
+        './PolylineOutlineMaterialProperty',
+        './PositionPropertyArray',
+        './RectangleGraphics',
+        './ReferenceProperty',
+        './SampledPositionProperty',
+        './ScaledPositionProperty',
+        './TimeIntervalCollectionProperty',
+        './WallGraphics'
+    ], function(
+        BoundingRectangle,
+        Cartesian2,
+        Cartesian3,
+        ClockRange,
+        ClockStep,
+        Color,
+        createGuid,
+        defaultValue,
+        defined,
+        defineProperties,
+        DeveloperError,
+        Ellipsoid,
+        Event,
+        getFilenameFromUri,
+        Iso8601,
+        JulianDate,
+        loadBlob,
+        loadXML,
+        CesiumMath,
+        NearFarScalar,
+        PinBuilder,
+        PolygonHierarchy,
+        Rectangle,
+        RuntimeError,
+        TimeInterval,
+        TimeIntervalCollection,
+        HorizontalOrigin,
+        LabelStyle,
+        Autolinker,
+        Uri,
+        when,
+        zip,
+        BillboardGraphics,
+        CompositePositionProperty,
+        ConstantPositionProperty,
+        DataSource,
+        DataSourceClock,
+        Entity,
+        EntityCollection,
+        LabelGraphics,
+        PathGraphics,
+        PointGraphics,
+        PolygonGraphics,
+        PolylineGraphics,
+        PolylineOutlineMaterialProperty,
+        PositionPropertyArray,
+        RectangleGraphics,
+        ReferenceProperty,
+        SampledPositionProperty,
+        ScaledPositionProperty,
+        TimeIntervalCollectionProperty,
+        WallGraphics) {
```

Comment:
A lot of these objects are used.  We should clean this up and only include needed items.